site/content/3.12/develop/http-api/schemas.md

```diff
@@ -0,0 +1,1116 @@
+---
+
+title: HTTP interface for database structures (schema, graph, view)
```

Comment:
This is ambiguous IMO. I generally wonder how to call it, because just Schemas can be confused with document schemas. Should we refer to it as "sampled schemas", "inferred schemas", or something similar?